api/client/beacon/client.go

```diff
@@ -272,7 +272,7 @@ type NodeVersion struct {
 	systemInfo     string
 }
 
-var versionRE = regexp.MustCompile(`^(\w+)\/(v\d+\.\d+\.\d+) \((.*)\)$`)
+var versionRE = regexp.MustCompile(`^(\w+)/(v\d+\.\d+\.\d+[-a-zA-Z0-9]*)\s*/?(.*)$`)
```

Comment:
fix for node version string parsing - adds a alphanumeric + dash character class to the capture before the option space/slash and system info. The old check was too brittle, expecting there to always be a space and parens.